cultivation/scripts/running/process_all_runs.py

```
@@ -3,17 +3,27 @@
 from pathlib import Path
 import subprocess
 import pandas as pd
+import sys
 from metrics import parse_gpx, run_metrics
```

Ruff (0.8.2)

`metrics.parse_gpx` imported but unused

Remove unused import

(F401)